manageiq-operator/api/v1alpha1/manageiq_types.go

```diff
@@ -1,16 +1,35 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
Why is this needed?

Reply:
Autogenerated.  I can remove it.

Reply:
https://www.apache.org/licenses/LICENSE-2.0#apply

Reply:
Ah yeah, the application to individual files is optional, but if it's autogenerated it's fine.  I mostly want to avoid keeping it up to date.